tests/unit/DateTimeRangeParserTest.php

```diff
@@ -0,0 +1,104 @@
+<?php namespace Core;
+
+use Carbon\Carbon;
+use Codeception\Test\Unit;
+use Gzero\Core\Parsers\DateTimeRangeParser;
+use Gzero\InvalidArgumentException;
+use Illuminate\Http\Request;
+
+class DateTimeRangeParserTest extends Unit {
+
+    /** @var UnitTester */
+    protected $tester;
+
+    /** @test */
+    public function isInstantiable()
+    {
+        $this->tester->assertInstanceOf(DateTimeRangeParser::class, new DateTimeRangeParser('date'));
+    }
+
+    /** @test */
+    public function itCanParseBetweenMatch()
+    {
+        $from = Carbon::parse('2021-05-02 0:43:31', 'Australia/Adelaide');
+        $to   = Carbon::parse('2021-05-01 23:43:31', 'America/New_York');
+
+        $parser = new DateTimeRangeParser('date');
+        $parser->parse(new Request(['date' => $from->toIso8601String() . "," . $to->toIso8601String()]));
+        $this->assertEquals('between', $parser->getOperation());
+        $result = $parser->getValue();
+
+        $this->assertEquals($from, $result[0]);
```

Comment:
Same here. I don't understand why these two date-times are equals. Are they in different timezone right?

Reply:
The same as above